Survey/Survey/SurveyView.swift

```diff
@@ -68,6 +66,6 @@ struct CustomButtonStyle: ButtonStyle {
 
 struct SurveyView_Previews: PreviewProvider {
     static var previews: some View {
-        SurveyView()
+        SurveyView(question: "How was your day?")
```

Comment:
Note that when I run the app I am not seeing the question show up. There are error messages around sandbox, not sure that's relevant: 

networkd_settings_read_from_file Sandbox is preventing this process from reading networkd settings file at "/Library/Preferences/com.apple.networkd.plist", please add an exception.
networkd_settings_read_from_file Sandbox is preventing this process from reading networkd settings file at "/Library/Preferences/com.apple.networkd.plist", please add an exception.
BOOL _NSPersistentUIDeleteItemAtFileURL(NSURL *const __strong) Failed to stat item: file:///Users/genepol/Library/Containers/Incredihire-LLC.Survey/Data/Library/Saved%20Application%20State/Incredihire-LLC.Survey.savedState/restorecount.plist